### ISSUE TITLE
Match standardjs for object-curly

### ISSUE THREAD

**Issue by mattkrick** [blakeembrey/tslint-config-standard#52]
standardjs allows spaces or no spaces for object-curlies (https://github.com/standard/eslint-config-standard/blob/master/eslintrc.json#L202). Would it be possible to follow suit for this, too?

**Comment by blakeembrey** [comment]
If possible, definitely.

**Comment by LinusU** [comment]
Standard now enforces spaces in object curlies, see standard/eslint-config-standard#124

**Comment by blakeembrey** [comment]
Yep, we should definitely enable it if it’s possible. Usually we don’t yet have the rules for it though.
